app/overlays/mtl/fpga_overlay.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=19200000
```

Comment:
Yes please.

Reply:
@abonislawski We will use FPGA platforms for CI.

Reply:
Why? It will fail for many tests

Reply:
Because for now it is our main tool for validation.
Until we move completely away from FPGA tests and have much more MTL silicon we should have this config simply to compare the results. Besides this will not be necessary when dynamic clock detection will be implemented so no reason to remove it now.

Reply:
@softwarecki we are going to use silicon for upstream CI, not FPGA.
That is why xtensa-build-zephyr.py mtl builds by default FW for RVP.
I kept fpga overlay for local builds so you can build locally for FPGA to compare results or for debugging purposes if you do not have access to silicon.